message-index/messages/GHC-03272/bidi-comment/after/BiDi.hs

```diff
@@ -0,0 +1,3 @@
+module BiDi where
+
+-- This comment contains bidirectional format chars
```

Comment:
```suggestion
-- This comment no longer contains bidirectional format chars
```

On one hand, I guess this doesn't make the diffs nice, but otherwise it's quite confusing as-is

Reply:
Ha! Good point!